content/posts/oas-swagger-ui.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Since this example is related to documentation, this should be placed within the `docs` directory instead. Also, I'd rename it as the API shown, in this case, "petstore" to illustrate better